MdeModulePkg/Library/PcdDatabaseLoaderLib/Dxe/PcdDatabaseLoaderLibDxe.c

```diff
@@ -0,0 +1,56 @@
+/** @file -- PcdDatabaseLoaderLibDxe.c
+
+  Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
```suggestion
  Copyright (c) Microsoft Corporation.
```
Please check in all the files.